src/components/experiment-wrapper/experiment-wrapper.js

```diff
@@ -40,12 +40,22 @@ const ExperimentWrapper = ({ theme }) => {
   );
 
   // Fetch all tracking data for selected runs.
-  const { data: { runTrackingData } = [], error: trackingDataError } =
+  const { data: { metric = [], json = [] } = [], error: trackingDataError } =
     useApolloQuery(GET_RUN_TRACKING_DATA, {
       skip: selectedRunIds.length === 0,
       variables: { runIds: selectedRunIds, showDiff: true },
     });
 
+  let runTrackingData = {};
+
+  if (metric.length > 0) {
+    runTrackingData['Metrics'] = metric;
+  }
+
+  if (json.length > 0) {
+    runTrackingData['JSON Data'] = json;
+  }
```

Comment:
I don't think this is needed as it recreates the shape of the data that comes directly from the query.